### DISCUSSION TITLE
jvm/jit

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#288] [Announcements]
# jvm/jit

本篇内容我们主要介绍了深入理解 JIT（即时编译器），包括 JVM 的编译器、JVM 的分层编译、JIT 的触发、JIT 的编译优化等内容。

https://javabetter.cn/jvm/jit.html

**Comment by chaofengdev** [comment]
这节太硬核了。看了半天没看懂，溜了。
